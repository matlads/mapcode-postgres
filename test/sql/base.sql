CREATE EXTENSION mapcode;

SELECT mapcode_version();

SELECT mapcode_encode(0.341637, 32.593781, 'UGA');

SELECT mapcode_decode('N7.FR', 'UGA');
