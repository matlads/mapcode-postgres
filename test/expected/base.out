CREATE EXTENSION mapcode;
SELECT mapcode_version();
 mapcode_version 
-----------------
 2.5.3
(1 row)

SELECT mapcode_encode(0.341637, 32.593781, 'UGA');
 mapcode_encode 
----------------
 UGA N7.FR
(1 row)

SELECT mapcode_decode('N7.FR', 'UGA');
   mapcode_decode   
--------------------
 0.341600,32.593770
(1 row)

